src/components/Footer/Footer.jsx

```diff
@@ -165,21 +165,21 @@ const Footer = ({ strings }) => (
               href="https://play.google.com/store/apps/details?id=com.opendota.mobile&hl=en"
               style={{ position: 'relative', left: '13px', top: '12px' }}
             >
-              <img src="/assets/images/google_play_store.png" alt="" height="46px" />
+              <img src="/assets/images/google_play_store.png" alt="download the android app on google play store" height="46px" />
             </a>
             <a
               href="https://itunes.apple.com/us/app/opendota/id1354762555?ls=1&mt=8"
               style={{ position: 'relative', left: '20px', top: '5px' }}
             >
-              <img src="/assets/images/apple_app_store.png" alt="" height="31px" />
+              <img src="/assets/images/apple_app_store.png" alt="download the iOS app on the app store" height="31px" />
             </a>
           </div>
         </div>
         <small className="about">
-          {strings.app_description}
+          <span id="app-description">{strings.app_description}</span>
           {' - '}
-          {strings.app_powered_by}
-          <a href="http://steampowered.com" target="_blank" rel="noopener noreferrer">
+          <span id="app-powered-by">{strings.app_powered_by}</span>
+          <a href="http://steampowered.com" aria-describedby="app-description app-powered-by" target="_blank" rel="noopener noreferrer">
```

Comment:
Links should include proper context for screen reader users. Visual users can see the description, but currently screen reader users only hear `http://steampowered.com`. Adding aria-describedby adds context for screen readers.